doc/howto/network_acls.md

```diff
@@ -359,14 +361,245 @@ lxc network acl show-log <ACL_name>
+
+(network-acls-edit-cli-rename)=
+### Rename an ACL via CLI
+
+Requirements:
+
+- You can only rename an ACL that is not currently assigned to a NIC or network. See the {ref}`network-acls-assign` section for more information.
+- The new name must meet the naming requirements listed in {ref}`network-acls-create`.
+
+To rename an ACL, query the following endpoint:
+
+```bash
+lxc network acl rename <old ACL name> <new ACL name>
+```
+
+
+(network-acls-edit-cli-properties)=
+### Edit other properties via CLI
+
+Run:
 
 ```bash
 lxc network acl edit <ACL_name>
 ```
 
-This command opens the ACL in YAML format for editing.
-You can edit both the ACL configuration and the rules.
+This command opens the ACL in YAML format for editing. You can edit any part of the configuration _except_ for the ACL name.
+
+````
+
+````{group-tab} API
+
+Use the API to:
+
+- {ref}`network-acls-edit-api-rename`
+- {ref}`network-acls-edit-api-properties`
+- {ref}`network-acls-edit-api-config`
+
+(network-acls-edit-api-rename)=
+### Rename an ACL via API
+
+Requirements:
+
+- You can only rename an ACL that is not currently assigned to a NIC or network. See the {ref}`network-acls-assign` section for more information.
+- The new name must meet the naming requirements listed in {ref}`network-acls-create`.
+
+To rename an ACL, query the following endpoint:
+
+```bash
+lxc query --request POST /1.0/network-acls/{name} --data '{
+  "name": "<new ACL name>"
+}'
+```
+
+See [the API reference](swagger:/network-acls/network_acl_post) for more information.
+
+#### Example
+
+Rename an ACL named `web-traffic` to `internal-web-traffic`:
+
+```bash
+lxc query --request POST /1.0/network-acls/web-traffic --data '{
+  "name": "internal-web-traffic"
+}'
+```
+
+(network-acls-edit-api-properties)=
+### Edit other properties via API
+
+To update any ACL property aside from `name`, query the following endpoint:
+
+```bash
+lxc query --request PUT /1.0/network-acls/{name} --data '{
+  "config": {
+    "user.<custom key name>": "<custom key value>"
+  },
+  "description": "<description of the ACL>",
+  "egress": [<egress rule>, <another egress rule...>,...],
+  "ingress": [<ingress rule>, <another ingress rule...>,...]
+}'
+```
+
+```{warning}
+Be careful! Any properties you omit from this request (aside from the ACL `name`) will be reset to defaults.
+```
+
+The `PUT` method used in this request performs a full replacement of the ACL's properties. With the exception of the `name` property, all other properties are overwritten by the data you provide. Any omitted properties are reset to default values. To preserve any specific properties when making an update, first {ref}`retrieve the current ACL properties <network-acls-show>`, then copy the values you want to keep into your request body. See [the API reference](swagger:/network-acls/network_acl_put) for more information.
+
+If you only need to add or update a `config.user.*` key, see: {ref}`network-acls-edit-api-config`.
+
+#### Example
```

Comment:
@minaelee can we avoid the need to repeat basic REST API fundamentals as an example for every API entity, and instead have an "API Basics" tutorial that we can link to from the warning?

Its starting to feel like these examples are being unnecessarily bloated and will be very repetitive across every API entity type we have. Can we cover the fundamental concept of "PUT replaces the entity's config, so you need to GET first and merge if you want to update" in a single place?